core/rfb.js

```diff
@@ -177,8 +179,14 @@ export default class RFB extends EventTargetMixin {
         this._keyboard.onkeyevent = this._handleKeyEvent.bind(this);
 
         this._mouse = new Mouse(this._canvas);
-        this._mouse.onmousebutton = this._handleMouseButton.bind(this);
-        this._mouse.onmousemove = this._handleMouseMove.bind(this);
+        this._mouse.onmousebutton = this._handlePointerPress.bind(this);
+        this._mouse.onmousemove = this._handlePointerMove.bind(this);
+
+        if (isTouchDevice) {
+            this._touch = new Touch(this._canvas, this._mouse);
+            this._mouse.ontouch = this._handlePointerPress.bind(this);
+            this._mouse.ontouchmove = this._handlePointerMove.bind(this);
```

Comment:
I believe these two should be `this._touch.ontouch` and `this._touch.ontouchmove`